submit-web/src/components/Submission/InfoBox/EntityInfoBox.tsx

```diff
@@ -36,31 +36,37 @@ export const EntityInfoBox = ({ submissionPackage }: InfoBoxProps) => {
         p: "16px 16px 16px 16px",
       }}
     >
+      <Grid item xs={12} md={6} container>
+        <Grid item xs={12}>
+          <Stack direction={"row"} spacing={2}>
+            <Typography color={BCDesignTokens.themeGray70}>
+              Condition:
+            </Typography>
+            <Typography color={"inherit"}>{condition}</Typography>
+          </Stack>
+        </Grid>
+
+        <Grid item xs={12}>
+          <Stack direction={"row"} spacing={2}>
+            <Typography color={BCDesignTokens.themeGray70}>
+              Supporting Conditions:
```

Comment:
may be need to have this same as in admin "Supporting Condition(s):" ?

Reply:
Done, thank you for letting me know